packages/cli/src/commands/org/config/monitor.ts

```diff
@@ -69,7 +71,7 @@ export default class Monitor extends Command {
     // TODO: revisit with a progress bar once the response contains the remaining resources to process
     const iteratee = (report: ResourceSnapshotsReportModel) =>
       this.refresh(report);
-    await snapshot.waitUntilDone(this.waitOptions, iteratee);
+    await snapshot.waitUntilDone(null, this.waitOption, iteratee);
```

Comment:
No, it is `null` because we don't want to wait for a particular operation.
If this is too confusing, the `operationToWaitFor` can be part of the options

Reply:
If it meant to be 'optional' (which, from what I understand, is the case), it should be marked as such ;)
Either by being later in the args and marked as optional,, either by being part of the waitOption obj, and marked as optional in the associated interface